### DISCUSSION TITLE
Node Support

### DISCUSSION THREAD

**Discussion by corinz** [panva/openid-client#519] [Q&A] [answered]
Are there plans to add node 17 support @panva?

`The engine "node" is incompatible with this module. Expected version "^12.19.0 || ^14.15.0 || ^16.13.0". Got "17.8.0"`

**Comment by panva** [comment, marked as answer]
No. https://github.com/panva/node-openid-client#install

The next supported engine will be node 18 active lts when it releases in october.

**Reply by corinz** [reply]
@panva Are there discussions/docs on the compatibility issues between this lib and node 17? Or is it simply untested?
